Update the first pages with a new image and restructure them (#161)
neo4j/docs-getting-started: modules/ROOT/pages/index.adoc

See the link:https://neo4j.com/docs/ops-manager/[Documentation] for Neo4j Ops Manager for more information.
* link:https://neo4j.com/docs/aura/current/getting-started/access-database/#_neo4j_data_importer/[Data Importer^] -- a no-code tool that allows you to load data from flat files (`.csv` and `.tsv`), define a graph model, map your data to it, and import it to your Neo4j database.
* link:https://neo4j.com/product/bloom/?ref=product[Neo4j Bloom^] -- visualization tool for business users that does not require any code or programming skills to view and analyze data.
See the link:https://neo4j.com/docs/bloom-user-guide/current/[Documentation^] for more information.

[[cypher]]
=== Cypher Query Language

link:https://neo4j.com/product/cypher-graph-query-language/?ref=product[Cypher] is an open data query language, based on the https://opencypher.org/?ref=cypher-web-page/[openCypher initiative].
It is the most established and intuitive query language used for property graphs.
Cypher can be characterized by following:

* easy to learn
* visual and logical
* secure, reliable, and data-rich
* open and flexible

> // Cypher is one of the graph query languages that are being used as the base for a new *standard* language -- link:https://www.gqlstandards.org/home[Graph Query Language].

Reviewer: We should definitely link to both the Introduction to Cypher chapter, and the Cypher documentation in this section

For more information about each of the components, go to the link:https://neo4j.com/product/[Product page^]. +
For more details about reference documentation, see xref:get-started-with-neo4j/index.adoc#[]. +
*The latest version of Neo4j is {neo4j-version-minor}.*

[[graph-database]]
=== Neo4j Graph Database

link:https://neo4j.com/product/neo4j-graph-database/?ref=product/[Neo4j Graph Database^] -- our core product, native graph database that is built to store and retrieve connected data. + 
link:https://neo4j.com/licensing/[Two editions^] -- a Community Edition and an Enterprise Edition -- are available. +
There are many ways to deploy Neo4j today: on-premises server installation, self-hosted in the cloud with pre-built images, or by simply using *Neo4j Aura*. +
You can find all necessary documentation on how to install and maintain Neo4j on the link:https://neo4j.com/docs/[Documentation page^]. 

[[neo4j-aura]]
=== Neo4j Aura

> Neo4j is a link:https://neo4j.com/cloud/[cloud-friendly] database, with a variety of cloud deployment options readily available.

Reviewer 2: ```suggestion
Neo4j is a link:https://neo4j.com/cloud/[cloud-friendly] database, with a variety of cloud deployment options available.
```

You can run Neo4j on public clouds like AWS, Azure, and Google Cloud Platform (GCP). +

> *Neo4j Aura* is a fully managed, cloud-native graph service, giving developers and data scientists the most advanced graph tech tools.

Reviewer 2: ```suggestion
*Neo4j Aura* is a fully managed, cloud-native graph service, that gives developers and data scientists advanced graph tech tools.
```

Neo4j Aura includes *AuraDB* and *AuraDS*.

> link:https://neo4j.com/cloud/aura/?ref=product[Neo4j AuraDB^] is the fully automated, always-on, and scalable graph database as a service. +

Reviewer 2: ```suggestion
link:https://neo4j.com/cloud/aura/?ref=product[Neo4j AuraDB^] is a graph database as a service. +
```

link:https://neo4j.com/cloud/platform/aura-graph-data-science/[Neo4j AuraDS] makes it easy to run graph algorithms on Neo4j by integrating two main components:

* **Neo4j Database**, where graph data are loaded and stored, and Cypher queries and all database operations (for example user management, query termination, etc.) are executed.
* **Graph Data Science**, a software component installed in the Neo4j Database, which main purpose is to run graph algorithms on in-memory projections of Neo4j Database.

Neo4j AuraDS is available on GCP. 
See link:https://neo4j.com/pricing/#graph-data-science[pricing] page for details.
You can find more details in the https://neo4j.com/docs/aura/aurads/[official documentation] for AuraDS. 

[[neo4j-gds]]
=== Neo4j Graph Data Science 

link:https://neo4j.com/product/graph-data-science/?ref=product[Neo4j Graph Data Science^] (GDS) gives you access to more than 65 graph algorithms, which can be executed with Neo4j and optimized for enterprise workloads and pipelines.
GDS helps you to get insights from big data in order to answer critical questions and improve predictions.

> GDS is the only connected data analysis platform that unifies the ML surface and graph database into a single workspace.

Reviewer 2: ```suggestion

```

> This way, data scientists run algorithms and ML models without jumping between tools for ETL.

Reviewer 2: ```suggestion

```

See link:https://neo4j.com/docs/graph-data-science/current/[documentation^] for the library for more details.

[[neo4j-tools]]
=== Neo4j Tools

Neo4j provides a variety of tools that aim to make learning and development of graph applications easier. 

> The link:https://neo4j.com/product/developer-tools/[Neo4j Developer Tools^] page introduces the most important of them.

Reviewer 2: That page only contains Desktop, Browser, and Data Importer, so I wouldn't say that it introduces the _most important_ dev tools. You could probably remove that line altogether as you're already introducing them here.

* link:https://neo4j.com/docs/desktop-manual/current/[Neo4j Desktop] -- a local development environment for working with Neo4j, whether using local database instances or databases located on remote servers.
Free download includes Neo4j Enterprise Edition license.
* link:https://neo4j.com/docs/browser-manual/current/[Neo4j Browser] -- online browser interface to query and view the data in the database.
Basic visualization capabilities using Cypher query language.
* link:https://neo4j.com/docs/ops-manager/[Neo4j Operations Manager] (NOM) -- a UI-based tool that enables a database administrator to monitor, administer, and operate all of the Neo4j database management systems in an Enterprise Edition. +
link:https://neo4j.com/videos/nom-bytes-1-are-we-good-get-an-estate-wide-view-of-managed-dbmss-with-neo4j-ops-manager/[Video series: _NOM Bytes_] introduces the product and provides some practical tips. +

> link:https://neo4j.com/docs/ops-manager/[Documentation] for Neo4j Ops Manager gives all information you may need for using it.    

Reviewer 2: ```suggestion
See the link:https://neo4j.com/docs/ops-manager/[Documentation] for Neo4j Ops Manager for more information.
```

> * link:https://neo4j.com/docs/aura/current/getting-started/access-database/#_neo4j_data_importer/[Data Importer^] -- a no-code tool that allows you to load data from flat files (`.csv` and `.tsv`) into your Neo4j databases, define a graph model, and map your data to it.

Reviewer 2: ```suggestion
* link:https://neo4j.com/docs/aura/current/getting-started/access-database/#_neo4j_data_importer/[Data Importer^] -- a no-code tool that allows you to load data from flat files (`.csv` and `.tsv`), define a graph model, map your data to it, and import it to your Neo4j database.
```

* link:https://neo4j.com/product/bloom/?ref=product[Neo4j Bloom^] -- visualization tool for business users that does not require any code or programming skills to view and analyze data.

> link:https://neo4j.com/docs/bloom-user-guide/current/[Documentation^] is also available on our Docs page.

Reviewer 2: ```suggestion
See the link:https://neo4j.com/docs/bloom-user-guide/current/[Documentation^] for more information.

[[cypher]]
=== Cypher Query Language

link:https://neo4j.com/product/cypher-graph-query-language/?ref=product[Cypher] is an open data query language, based on the https://opencypher.org/?ref=cypher-web-page/[openCypher initiative].

> It is the most established and intuitive query language to learn with property graphs.

Reviewer 2: ```suggestion
It is the most established and intuitive query language used for property graphs.
```

Cypher can be characterized by following:

* easy to learn
* visual and logical
* secure, reliable, and data-rich
* open and flexible

// Cypher is one of the graph query languages that are being used as the base for a new *standard* language -- link:https://www.gqlstandards.org/home[Graph Query Language].

[[connect-to-neo4j]]
=== Connecting to Neo4j

> It is important for us to provide a wide range of opportunities for integrating Neo4j with any working environment. 

Reviewer 2: ```suggestion
Neo4j provides a wide range of opportunities for integrating Neo4j with any working environment. 
```

* Third, **Neo4j Graph Data Science** is a connected data analytics and machine learning platform that helps you understand the connections in big data.

++++
<div class="responsive-embed">
<iframe width="680" height="425" src="https://www.youtube.com/embed/urO5FyP9PoI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
</div>
++++

////

== Neo4j Community Forum

> Neo4j is dedicated to kind and open communication and aims to create developer-friendly environment for everybody. +

Reviewer 2: ```suggestion
Neo4j is dedicated to kind and open communication and aims to create a developer-friendly environment. +
```

> Neo4j is open-source and has the largest and most vibrant community of graph database enthusiasts. 

Reviewer 2: ```suggestion
Neo4j is open-source and has a large and vibrant community of graph database enthusiasts. 
```

You can reach out to our link:https://community.neo4j.com/[Community Forum] or link:https://discord.com/invite/neo4j[Discord Chat] for any help or advice you may need. +

== Neo4j GraphAcademy

> If you want learn how to build, optimize, and launch your Neo4j project, visit our link:https://graphacademy.neo4j.com/[GraphAcademy page].

Reviewer 2: ```suggestion
If you want learn how to build, optimize, and launch your Neo4j project, visit the link:https://graphacademy.neo4j.com/[GraphAcademy page].
```

GraphAcademy courses are free, interactive, and hands-on. +

> All of them have been developed by Neo4j professionals with years of experience.

Reviewer 2: ```suggestion
They have all been developed by experienced Neo4j professionals.
```

> Our set of courses aims a wide range of job role: data scientists, developers, and database administrators.

Reviewer 2: ```suggestion
Our set of courses targets a wide range of job roles: data scientists, developers, and database administrators.
```

> To help you to get more information about Neo4j products and projects, we would like to provide you with the xref::appendix/getting-started-resources.adoc[Getting started resources], which may inspire you to explore different possibilities of graph technologies. +

Reviewer 2: ```suggestion
For more information about Neo4j products and projects, see the xref::appendix/getting-started-resources.adoc[Getting started resources]. +
```
